dataset/src/main/scala/frameless/ops/GroupByOps.scala

```diff
@@ -82,6 +82,11 @@ class GroupedByManyOps[T, TK <: HList, K <: HList, KT](
   private def retainGroupColumns: Boolean = {
     self.dataset.sqlContext.getConf("spark.sql.retainGroupColumns", "true").toBoolean
   }
+
+  def pivot[P: CatalystPivotable, Values <: HList](pivotColumn: TypedColumn[T, P],
+                                values: Values)(
```

Comment:
V is already used `class GroupedBy1Ops[K1, V]`. That's why I used `Values`. It's also more expressive. We are using V usually to express the type of a dataset (`TypedDataset[V]`).